File: Data Structures & Algorithms/merge-strings-alternately/submission-0.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int i = 1;
        int j = 0;
        while(i < word1.length() && j<word2.length()){
            word1.insert(i,1, word2[j]);
            j++;
            i+=2;
        }

        while(j<word2.length()){
            word1.push_back(word2[j++]);
        }

        return word1;
    }
};

File: Data Structures & Algorithms/plus-one/submission-0.py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        n = len(digits)

        for i in range(len(digits)):
            if digits[n-i-1] < 9:
                digits[n-i-1] += 1
                return digits
            digits[n-i-1] = 0
        digits.insert(0,1)
        return digits
        